Workbooks/CosmosDb/Resource Insights/Resource Insights.workbook

```diff
@@ -6,50 +6,20 @@
       "content": {
         "version": "KqlParameterItem/1.0",
         "parameters": [
-          {
-            "id": "1a5523c4-8b8f-405b-915b-29f2bb09ad6b",
-            "version": "KqlParameterItem/1.0",
-            "name": "Subscription",
-            "label": "Subscriptions",
-            "type": 6,
-            "description": "All subscriptions with CosmosDB accounts",
-            "isRequired": true,
-            "query": "where type =~ 'Microsoft.DocumentDB/databaseAccounts'\r\n\t| summarize Count = count() by subscriptionId\r\n\t| order by Count desc\r\n\t| extend Rank = row_number()\r\n\t| project value = subscriptionId, label = subscriptionId, selected = Rank == 1",
-            "crossComponentResources": [
-              "value::selected"
-            ],
-            "value": "",
-            "typeSettings": {
-              "additionalResourceOptions": [],
-              "showDefault": false
-            },
-            "timeContext": {
-              "durationMs": 86400000
-            },
-            "queryType": 1,
-            "resourceType": "microsoft.resourcegraph/resources"
-          },
           {
             "id": "aa83f370-5c47-4eb5-b44a-fedcad97f71b",
             "version": "KqlParameterItem/1.0",
             "name": "Resources",
```

Comment:
so this is intended to be an "at resource" template, but is powered by being fed a "Resources" parameter.  this isn't how at resource templates usually work, is it?  this means when they write code to open this blade, they'll also have to pass this as a parameter in the NotebookParams field when they open the workbooks blade with this template?  this will NOT work if this item appears in a gallery, nothign will ever set the param!

Reply:
yves showed that this IS working and that in his example he is passing the resource as component id.

Reply:
for now this is on all resource templates so I thought it was standard. Clicking on the template from the gallery opens the workbook with the right resource